app/controllers/alerts_list_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 # Controller for showing Alert records
 class AlertsListController < ApplicationController
+  include AlertsHelper
```

Comment:
@sejalkhatri  I am not sure. **Is this a new behaviour in Rails 5?** 

AFAIK, the helper file which matches the name of the controller is only included in the controller.

For eg: If the controller name is `AlertsListController`, `AlertsListHelper` is automatically included for you.